README.md

```diff
@@ -69,19 +72,34 @@ bluesky-comments:
   filter-empty-replies: true
   n-show-init: 3
   n-show-more: 2
+  n-show-depth: 3
 ---
 ```
 
+Each of these options can also be set directly for a single comments section by providing the option as an inline attribute in the shortcode, e.g. `{{{< bluesky-comments n-show-init=3 >}}}`. These values take precedence over the above settings.
+
 ### Available Options
 
-- `mute-patterns`: An array of strings or regex patterns (enclosed in `/`) to filter out comments containing specific text
-- `mute-users`: An array of Bluesky DIDs to filter out comments from specific users
-- `filter-empty-replies`: Boolean flag to filter out empty or very short replies, including bookmarking (📌) replies (default: `true`)
-- `n-show-init`: Number of top-level comments to show initially (default: `3`)
-- `n-show-more`: Number of replies to reveal when the user clicks on the "Show more" button (default: `2`)
-- `header`: Whether or not to add a level-2 header above the comments. Use `header="true"` as a shortcut for `header="Comments"` (default: `false`).
+`mute-patterns`
```

Comment:
Should we place this inside of a table for consistency? (I agree the bullet-point list didn't look good here w/ more options.)

Reply:
yeah, good point. Though I'd rather cut out everything but installation and basic usage from the README. I don't like updating things in two places (too easy to forget one of them). As long as the README gives a good sense of what the extension does, I'd rather push people to the website for more detailed documentation like configuration, etc.

Reply:
@gadenbuie not a fan either. Majority of the `README.md` should be matriculated over to the main website. I'll handle this clean up if you want.

Reply:
Thanks! I took a quick pass at it in #42. Mostly I just cut out content that I know we already have on the website. Feel free to add to that PR!